src/FeatureUtil/FeatureUtil.js

```diff
@@ -22,7 +23,29 @@ class FeatureUtil {
     let featureId = feature.getId();
     let featureIdParts = featureId ? featureId.split('.') : featureId;
 
-    return Array.isArray(featureIdParts) ? featureIdParts[0] : undefined;
+    return isArray(featureIdParts) ? featureIdParts[0] : undefined;
+  }
+
+  /**
+   * Extracts the featureType name from given GetFeatureInfo URL.
+   * This method is mostly useful for raster layers which features could have
+   * no ID set.
+   *
+   * @param {string} url GetFeatureInfo URL possibly containing featureType name.
+   *
+   * @return {string} Obtained featureType name as string.
+   */
+  static getFeatureTypeNameFromGetFeatureInfoUrl(url) {
+    const regex = /query_layers=(.*?)&/i;
+    let match = url.match(regex);
+    let featureTypeName;
+    if (isArray(match) && match[1]) {
```

Comment:
No need for `isArray` since match always either returns an arra or null. This could then be 

```js
if  (match && match[1]) {
  // ...
}
```